_docker/drupal/drupal-filesystem/web/themes/custom/rhdp/rhd-frontend/src/styles/_books.scss

```diff
@@ -97,7 +97,7 @@ p.desc {
 @include mobile-landscape-and-down {
 
 .book {
-  padding-right:0px 
+  padding-right:0px
```

Comment:
It may not be required here since there's only one line, but consistency with the trailing semi-colon would be good.